artison.py

```diff
@@ -38,9 +37,9 @@ def load_packer_file(self, filepath):
             self.packer_file = json.load(f)
 
     def _compile_builders(self, builder):
-        if not self.config.get('builders'):
```

Comment:
If the `builders` section is ommitted, self.config['builders'] will throw an exeception, where as self.config.get('builders') will just return `None`. I think we should keep `get`